src/IJulia.jl

```diff
@@ -211,6 +211,34 @@ a subset of cell inputs/outputs to clear.
 clear_history
 
 #######################################################################
+# methods to print history or any subset thereof
+function history(io::IO, indices::AbstractVector{Int})
+    for n in intersect(indices, 1:IJulia.n)
+      if haskey(In, n)
+        print(In[n])
+      end
+    end
+end
+
+history(io::IO, x::Union{Integer,AbstractVector{Int}}...) = history(io, vcat(x...))
+history(x...) = history(STDOUT, x...)
+history(io::IO, x...) = throw(MethodError(history, (x...)))
+history() = history(1:n)
+"""
+    history([io], [indices])
+
+The `history()` function prints all of the input history stored in
+the running IJulia notebook in a format convenient for copying.
+
+The optional `indices` argument is a collection of indices indicating
```

Comment:
`is one or more indices or collections of indices indicating a subset input cells to print`